Source/Diagnostics/ReducedDiags/FieldProbe.cpp

```diff
@@ -153,24 +184,48 @@ FieldProbe::FieldProbe (std::string rd_name)
 
 void FieldProbe::InitData ()
 {
-    //create 1D array for X, Y, and Z of particles
-    amrex::Vector<amrex::ParticleReal> xpos;
-    amrex::Vector<amrex::ParticleReal> ypos;
-    amrex::Vector<amrex::ParticleReal> zpos;
-
-    // for now, only one MPI rank adds a probe particle
-    if (ParallelDescriptor::IOProcessor())
+    if (dimension == 0)
     {
-        xpos.push_back(x_probe);
-        ypos.push_back(y_probe);
-        zpos.push_back(z_probe);
+        //create 1D array for X, Y, and Z of particles
+        amrex::Vector<amrex::ParticleReal> xpos;
+        amrex::Vector<amrex::ParticleReal> ypos;
+        amrex::Vector<amrex::ParticleReal> zpos;
+
+        // for now, only one MPI rank adds a probe particle
+        if (ParallelDescriptor::IOProcessor())
+        {
+            xpos.push_back(x_probe);
+            ypos.push_back(y_probe);
+            zpos.push_back(z_probe);
+        }
+
+        // add particles on lev 0 to m_probe
+        m_probe.AddNParticles(0, xpos, ypos, zpos);
     }
+    else
+    {
+        amrex::Vector<amrex::ParticleReal> xpos(resolution, 0);
+        amrex::Vector<amrex::ParticleReal> ypos(resolution, 0);
+        amrex::Vector<amrex::ParticleReal> zpos(resolution, 0);
```

Comment:
@TiberiusRheaume as discussed last week, please fix this. Same logic as for one particle: we don't want to initialize the same particle N times if we have N MPI ranks.